presto-main/src/main/java/com/facebook/presto/execution/StageStateMachine.java

```diff
@@ -236,6 +237,7 @@ public StageInfo getStageInfo(Supplier<Iterable<TaskInfo>> taskInfosSupplier, Su
 
             cumulativeMemory += taskStats.getCumulativeMemory();
             totalMemoryReservation += taskStats.getMemoryReservation().toBytes();
+            totalRevocableMemoryReservation += taskStats.getRevocableMemoryReservation().toBytes();
```

Comment:
`totalMemoryReservation` name suggests that it includes the `revocable` memory. Consider renaming `totalMemoryReservation` to sth other.